source/Octopus.Tentacle.Client/ITentacleClient.cs

```diff
@@ -5,13 +5,17 @@
 using Octopus.Diagnostics;
 using Octopus.Tentacle.Client.Scripts;
 using Octopus.Tentacle.Contracts;
-using Octopus.Tentacle.Contracts.ScriptServiceV2;
 using Octopus.Tentacle.Contracts.ScriptServiceV3Alpha;
 
 namespace Octopus.Tentacle.Client
 {
     public interface ITentacleClient : IDisposable
     {
+        Task<CommandResponseV3Alpha> StartScript(StartScriptCommandV3Alpha startScriptCommand, ILog logger, CancellationToken scriptExecutionCancellationToken);
```

Comment:
Further to @LukeButters point around service types leaking out of tentacle client, The `StartScriptCommandV3Alpha` also really needs to get changed to a non-service type.

In my mini-RFC (as mentioned by Pawel), part of the proposal is to remove service types leaking into the Server codebase.

I'm happy for you to leave it for now, but just be aware that this is something I'm intending to address as part of the proposed solution in the RFC

Reply:
Sounds good. When you're back, maybe we should get together to sync our two streams of work so we get a good path forward 🙂